### PR TITLE
Fargate bmlt

### PR DESCRIPTION
**Pull request by pjaudiomv**

### CODE REVIEW

File: terraform/cloudwatch.tf [hunk 1]
```diff
@@ -59,20 +59,14 @@ resource "aws_cloudwatch_event_target" "root_server_import" {
...
 
   ecs_target {
+    launch_type         = "FARGATE"
     task_count          = 1
     task_definition_arn = aws_ecs_task_definition.tomato_root_server_import.arn
+
+    network_configuration {
+      security_groups  = [aws_security_group.cluster.id]
+      subnets          = [aws_subnet.public_a.id, aws_subnet.public_b.id]
+      assign_public_ip = true
```
Comment (jbraswell) [inline terraform/cloudwatch.tf:69]: ecs targets shoudn't have public ips

Comment (jbraswell) [reply terraform/cloudwatch.tf:69]: they're behind a load balancer, you don't hit them directly

File: terraform/ecs-service.tf [hunk 2]
```diff
@@ -47,19 +50,29 @@ EOF
...
     "volumesFrom": [],
     "extraHosts": null,
     "dnsServers": null,
     "disableNetworking": null,
+    "networkMode": "awsvpc",
     "dnsSearchDomains": null,
     "portMappings": [
       {
-        "hostPort": 0,
+        "hostPort": 8000,
```
Comment (jbraswell) [inline terraform/ecs-service.tf:75]: why is hostport required? normally you leave this at 0 and it assigns a random port

File: terraform/cloudwatch.tf (hunk 1 above)
Comment (jbraswell) [reply terraform/cloudwatch.tf:69]: what is associating the tasks with the load balancer? are you doing that manually?

File: terraform/ecs-service.tf (hunk 2 above)
Comment (pjaudiomv) [reply terraform/ecs-service.tf:75]: Because fargate I think we can just remove that

**Comment by pjaudiomv** [comment]
Driving to Columbia feel free to make changes to the branch

File: terraform/ecs-service.tf (hunk 2 above)
Comment (pjaudiomv) [reply terraform/ecs-service.tf:75]: 5C390C5B-BE09-4819-8BE2-F05B4F67E65F

Comment (pjaudiomv) [reply terraform/ecs-service.tf:75]: https://aws.amazon.com/blogs/compute/migrating-your-amazon-ecs-containers-to-aws-fargate/

File: terraform/cloudwatch.tf (hunk 1 above)
Comment (pjaudiomv) [reply terraform/cloudwatch.tf]: It gets attached here https://github.com/bmlt-enabled/tomato/blob/183276f4145f5d7214597153ce8e60aca9097ef8/terraform/ecs-service.tf#L222

**Pull request closed without merging**